discover/portal/articles/140-workflow/03-managing-workflows.markdown

```diff
@@ -0,0 +1,149 @@
+# Managing Workflows
+
+Like other resources, workflow definitions can be added, edited, and deleted.
+But that's just the beginning of workflow management.
+
+- Because they can be complex works in progress, workflow definition versioning
+is supported.
+
+- Unpublished drafts can be saved.
+
+- Because workflow definitions are XML files, they're portable. Thus, they can also be uploaded. 
+
+Start by learning about basic workflow management.
+
+## Adding, Editing, and Deleting
+
+To add a workflow definition,
+
+1.  Go to Control Panel &rarr; Configuration &rarr; Workflow.
+
+2.  In the default view (Workflows), click the Add button
+    (![Add](../../../images/icon-add.png)).
+
+3.  From here you're either 
+    [writing an XML definition](/develop/tutorials/-/knowledge_base/7-1/crafting-xml-workflow-definitions), 
+    [designing a definition in Kaleo Designer (DXP only)](/discover/portal/-/knowledge_base/7-1/kaleo-designer), 
+    or uploading an existing definition ([see
+    below](#uploading-workflow-definitions)).
+
+To edit a definition, 
+
+1.  Go to Control Panel &rarr; Configuration &rarr; Workflow.
+
+2.  Click the Actions button (![Actions](../../../images/icon-actions.png)) for
+    the workflow to edit, and click *Edit*.
+
+To delete a definition,
+
+1.  Go to Control Panel &rarr; Configuration &rarr; Workflow.
+
+2.  Click the Actions button (![Actions](../../../images/icon-actions.png)) for
+    the workflow to edit, and click *Unpublish*.
+
+    Published workflows cannot be deleted, so you must unpublish the workflow
+    definition first. 
+
+    You can't unpublish a definition if it's activated for an asset. First
+    dissociate the workflow definition from any assets that use it. See
+    [here](/discover/portal/-/knowledge_base/7-1/activating-workflow) for more
+    information.
+
+## Uploading Workflow Definitions [](id=uploading-workflow-definitions)
+
+If you have an XML definition file locally (perhaps one of the embedded
+workflows), upload it to Liferay Portal:
```

Comment:
@rbohl I'm not familiar with embedded workflows. Does an example definition really need to be mentioned? Or is there an article we can link to about the embedded workflows?